Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -512,8 +512,7 @@ then
     else echo "Unsupported/unknown product: ${PRODUCT_NAME}"
     fi
 elif [[ "${WK_PLATFORM_NAME}" == iphoneos ||
-        "${WK_PLATFORM_NAME}" == appletvos ||
-        "${WK_PLATFORM_NAME}" == watchos ]]
+        "${WK_PLATFORM_NAME}" == appletvos ]]
```

Comment:
Is is possible to check ${WK_PLATFORM_NAME}" == watchOS inside ios_family_process_webcontent_entitlements? If so, perhaps that would be a smaller change?

Reply:
Looks like that will work!